examples/tests/sdk_tests/bucket/signed_url.test.w

```diff
@@ -1,29 +1,72 @@
 bring cloud;
-bring http;
+bring fs;
 bring util;
+bring expect;
 
-let testBucket = new cloud.Bucket(public: true) as "testBucket";
+let bucket = new cloud.Bucket();
 
-test "signedUrl" {
-  let var error = "";
-  testBucket.put("file1.txt", "Foo");
-  
-  if (util.env("WING_TARGET") != "sim") { 
-    let signedUrl = testBucket.signedUrl("file1.txt");
-    assert(http.get(signedUrl).body ==  "Foo"); 
-  } 
+test "signedUrl GET" {
+  let KEY = "tempfile.txt";
+  let VALUE = "Hello, Wing!";
+
+  bucket.put(KEY, VALUE);
+
+  let getSignedUrl = bucket.signedUrl(KEY);
+
+  // Download file from private bucket using GET presigned URL
+  let output = util.shell("curl \"{getSignedUrl}\"");
```

Comment:
interesting, does it work the same on Windows, Linux and Mac?

Reply:
All tests passed so it looks like that.